iree-tf/benchmark/benchmark_model.py

```diff
@@ -28,241 +29,229 @@
 
 
 def benchmark_lookup(unique_id: str):
-    if unique_id not in tf_model_definitions.TF_MODELS_DICT:
-        raise ValueError(f"Id {unique_id} does not exist in model suite.")
+  if unique_id not in tf_model_definitions.TF_MODELS_DICT:
+    raise ValueError(f"Id {unique_id} does not exist in model suite.")
 
-    model_definition = tf_model_definitions.TF_MODELS_DICT[unique_id]
-    if unique_id.startswith(unique_ids.MODEL_RESNET50_FP32_TF):
-        return ("RESNET50", resnet50.ResNet50, model_definition.input_batch_size)
-    elif unique_id.startswith(unique_ids.MODEL_BERT_LARGE_FP32_TF):
-        return ("BERT_LARGE", bert_large.BertLarge, model_definition.input_batch_size)
-    elif unique_id.startswith(unique_ids.MODEL_T5_LARGE_FP32_TF):
-        return ("T5_LARGE", t5_large.T5Large, model_definition.input_batch_size)
-    else:
-        raise ValueError(f"Model definition not supported")
+  model_definition = tf_model_definitions.TF_MODELS_DICT[unique_id]
+  if unique_id.startswith(unique_ids.MODEL_RESNET50_FP32_TF):
+    return ("RESNET50", resnet50.ResNet50, model_definition)
```

Comment:
(No need to address here): IIUC here we still download the (latest?) models from huggingface. Maybe we can load from the saved models in the ModelArtifact?

Reply:
Good point. We may want to freeze the huggingface and keras versions in the pip requirements.txt file. I think loading the SavedModels will only load the frozen weights and not necessarily any tweaks made to the model architecture. Let's put an action item to handle this - could be fixing dependencies to a specific version/sha, creating a docker image with the venv already setup, etc.